src/creature.h

```diff
@@ -544,6 +544,9 @@ class Creature
         virtual const std::string &symbol() const = 0;
         virtual bool is_symbol_highlighted() const;
 
+		// Turn since creature was milked for last time
+		int turn_next_milking = 0;
```

Comment:
The comment and the member name seem to disagree: is this the time of the last (most recent) milking (always in the past) or is it the time of the next milking (in the future)?